### PR TITLE
Release settings

### PR DESCRIPTION
**Pull request by schleichardt**
@lauraluiz @dariuszor please review

### CODE REVIEW

**Comment by dariuszor** [comment]
PR: 👍 
During reading code I realized we have CmsIdentifier class, which is not used anymore.
Should I create separate PR for removing it?

File: project/Release.scala [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    pomIncludeRepository := { _ => false },
+    pomExtra := (
+      <scm>
+        <url>git@github.com:scommercetools/commercetools-sunrise-java.git</url>
+        <connection>scm:git:git@github.com:commercetools/commercetools-sunrise-java.git</connection>
+      </scm>
+        <developers>
+          <developer>
+            <id>dariuszor</id>
+            <name>dariuszor</name>
```
Comment (lauraluiz) [inline project/Release.scala]: @dariuszor any name you would like to put in here? :)

**Comment by lauraluiz** [comment]
@dariuszor I'll take care of a last review before the release, don't worry

File: project/Release.scala (hunk 1 above)
Comment (dariuszor) [inline project/Release.scala]: Dariusz Orlowski :)

**Comment by lauraluiz** [comment]
Solved https://github.com/commercetools/commercetools-sunrise-java-cms/pull/14 https://github.com/commercetools/commercetools-sunrise-java-cms/pull/15 https://github.com/commercetools/commercetools-sunrise-java-cms/pull/16

**Pull request closed without merging**